x/leverage/simulation/operations.go

```diff
@@ -312,50 +326,50 @@ func randomCoin(r *rand.Rand, coins sdk.Coins) sdk.Coin {
 // randomSpendableFields returns an random account and coins to spend in the simulation.
 func randomSpendableFields(
 	r *rand.Rand, ctx sdk.Context, accs []simtypes.Account, bk types.BankKeeper,
-) (acc simtypes.Account, spendableCoin sdk.Coin, skip bool) {
+) (acc simtypes.Account, spendableToken sdk.Coin, skip bool) {
 	acc, _ = simtypes.RandomAcc(r, accs)
 
 	spendableBalances := bk.SpendableCoins(ctx, acc.Address)
 
-	spendableCoins := simtypes.RandSubsetCoins(r, spendableBalances)
-	if spendableCoins.Empty() {
+	spendableTokens := simtypes.RandSubsetCoins(r, spendableBalances)
+	if spendableTokens.Empty() {
 		return acc, sdk.Coin{}, true
 	}
 
-	return acc, randomCoin(r, spendableCoins), false
+	return acc, randomCoin(r, spendableTokens), false
 }
 
 // randomCollateralFields returns an random account and coins to withdraw in the simulation.
```

Comment:
// randomCollateralFields returns a random account and an sdk.Coin from its collateral.
// It returns skip=true if no collateral is found.